src/overlays/actors/ovl_En_Torch2/z_en_torch2.c

```diff
@@ -19,18 +31,137 @@ const ActorInit En_Torch2_InitVars = {
     (ActorFunc)EnTorch2_Init,
     (ActorFunc)EnTorch2_Destroy,
     (ActorFunc)EnTorch2_Update,
-    (ActorFunc)EnTorch2_Draw
+    (ActorFunc)EnTorch2_Draw,
+};
+
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_METAL, AT_NONE, AC_ON | AC_HARD | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_PLAYER | OC1_TYPE_1 | OC1_TYPE_2,
+      OC2_TYPE_2, COLSHAPE_CYLINDER },
+    { ELEMTYPE_UNK2, { 0x00100000, 0, 0 }, { 0xF7CFFFFF, 0, 0 }, TOUCH_NONE, BUMP_ON | BUMP_HOOKABLE, OCELEM_ON },
+    { 20, 60, 0, { 0, 0, 0 } },
+};
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_S8(colChkInfo.mass, MASS_IMMOVABLE, ICHAIN_STOP),
+};
+
+// Statues for each of Link's different forms
+// (Playing elegy as Fierce Deity puts down a human statue)
+static Gfx* const sGfxs[] = {
```

Comment:
Since you know what the display lists are, the name `sGfxs` feels too generic. You could for instance name it `sStatueDLists` or similar.